www/js/survey/multilabel/MultiLabelButtonGroup.tsx

```diff
@@ -55,7 +55,7 @@ const MultilabelButtonGroup = ({ trip, buttonsInline=false }) => {
     if (isOther) {
       /* Let's make the value for user entered inputs look consistent with our other values
        (i.e. lowercase, and with underscores instead of spaces) */
-      chosenLabel = chosenLabel.toLowerCase().replace(" ", "_");
+       chosenLabel = readableLabelToKey(chosenLabel);
```

Comment:
This is fine here, but a quick note for https://github.com/e-mission/e-mission-phone/commit/c480d67217eb18cf7c9fe24595a3bec04a9dc69e we should only use this for "other" labels. For the labels that have partner-provided readable labels, we should use them instead - e.g. `drove_alone` should become `Gas Car Drove Alone`, not `Drove Alone`

Reply:
I see. It will need to be changed on the new Dashboard, then